uptime/gen/gen.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+// TODO: man, how this works with local package?
+
+import (
+	gen "github.com/whyrusleeping/cbor-gen"
+
+	"github.com/filecoin-project/lotus/uptime"
+)
+
+func main() {
+	if err := gen.WriteMapEncodersToFile("../cbor_gen.go", "uptime",
+		// uptime.NodeInfo{},
```

Comment:
Careful, this is commented. Is this on purpose?

Reply:
If you change some field that causes an error on the package, cbor_gen will fail. What I usually do is to keep the interface implementation, and remove all the code and just return `nil` so it compiles without failure and you can re-generate the code.